pkgs/applications/emulators/wine/sources.nix

```
@@ -12,6 +12,15 @@ let fetchurl = args@{url, sha256, ...}:
   pkgs.fetchFromGitHub { inherit owner repo rev sha256; } // args;
     fetchFromGitLab = args@{domain, owner, repo, rev, sha256, ...}:
   pkgs.fetchFromGitLab { inherit domain owner repo rev sha256; } // args;
+
+    updateScriptPreamble = ''
+      set -eou pipefail
+      PATH=${with pkgs; lib.makeBinPath [ common-updater-scripts jq ]}:$PATH
```

```suggestion
      PATH=${with pkgs; lib.makeBinPath [ common-updater-scripts jq ]}
```
We shouldn't leak the PATH in